gatsby-plugin-webfonts/src/web-fonts.js

```diff
@@ -4,10 +4,12 @@ import postcss from "postcss";
 import cssnano from "cssnano";
 
 import google from "./modules/google";
+import selfHosted from "./modules/selfHosted";
 
 export default async function webFonts(options) {
   const modules = {
     google: google(options),
+    selfHosted: selfHosted(),
```

Comment:
The options have to be passed here. For example for the pathprefix...
See https://github.com/hupe1980/gatsby-plugin-webfonts/tree/master/e2e-tests/path-prefix-prod-runtime